xml/article_nfs_storage.xml

```diff
@@ -714,21 +694,58 @@ include /etc/drbd.d/*.res;</screen>
 
   <para>In its simplest form, the command to mount the NFS export looks like
    this:</para>
-  <screen>&prompt.root;<command>mount -t nfs &nfs-vip-exports;:/srv/nfs/work /home/work</command></screen>
+  <screen>&prompt.root;<command>mount -t nfs &nfs-vip-exports;:/srv/nfs/exportfs /home/exportfs</command></screen>
   <para>
    To configure a specific transport protocol (<option>proto</option>)
    and maximum read and write request sizes (<option>rsize</option> and
     <option>wsize</option>), use:
   </para>
   <screen>&prompt.root;<command>mount -o rsize=32768,wsize=32768 \
-    &nfs-vip-exports;:/srv/nfs/work /home/work</command></screen>
+    &nfs-vip-exports;:/srv/nfs/exportfs /home/exportfs</command></screen>
   <para>
-   In case you need to be compatible with NFS version&nbsp;3, include the value
+   If you need to be compatible with NFS version&nbsp;3, include the value
    <option>vers=3</option> after the <option>-o</option> option.
   </para>
   <para>
-   For further NFS mount options, consult the <command>nfs</command> man page.
+   For further NFS mount options, refer to the <command>nfs</command> man page.
   </para>
+  <warning>
+   <title>Low lease time can cause loss of file state</title>
+   <para>
+    NFS clients regularly renew their state with the NFS server. If the lease time
+    is too low, operating system or network delays can cause the timer to expire
+    before the renewal is complete, leading to loss of file state and I/O errors.
+   </para>
+   <para>
+    <literal>NFSV4LEASETIME</literal> is set on the NFS server in the file
+    <filename>/etc/sysconfig/nfs</filename>. The default is 90 seconds.
+    If lowering the lease time is necessary, we recommend a value of 60 or
+    higher. We strongly discourage values lower than 30.
+   </para>
+  </warning>
```

Comment:
I've put this note here in Chapter 7 Using the NFS service, as I originally suggested, but maybe it's better in Section 4.3 Creating the file system? Let me know what you think.

I also skimmed through the Admin guide and didn't notice anywhere that seemed like a good place to also add this note, but I'm open to suggestions.

Reply:
> I've put this note here in Chapter 7 Using the NFS service, as I originally suggested, but maybe it's better in Section 4.3 Creating the file system? Let me know what you think.
> 

However, I prefer to move this WARNING under "primitive p_nfsserver nfsserver".  Simply because  NFSV4LEASETIME in `/etc/sysconfig/nfs` is directly used by the nfs server, instead of the nfs client.

Reply:
That makes sense to me too. @dpartrid what do you think?

Reply:
Yes, it looks good.  (And this was indeed within the PDF I reviewed.)